README.md

```diff
@@ -50,6 +50,51 @@ Range from `0.0` to `1.0`.
 
 Entity hash, of the door, window, etc.
 
+## Ymt full entries
```

Comment:
I don't know if I feel like all of this is required to be honesty, just stating that 'If there are no attached objects to the portal, leave the PortalEntityList empty' would suffice

Reply:
Thats true. Ill edit it now